src/server.ts

```diff
@@ -42,11 +45,17 @@ export function getServer(
 
   // Set request-specific values in the very first middleware.
   app.use('/*', (req, res, next) => {
+    // Set function context in app locals for the lifetime of the request.
+    res.locals.context = context || {};
+
     setLatestRes(res);
     res.locals.functionExecutionFinished = false;
     next();
   });
 
+  // Use OpenFunction middlewares
+  app.use(daprBindingOutputMiddleware);
```

Comment:
In sync function, users can also send output to dapr pubsub, not just output binding

Reply:
Yes, the implemention already supports pub, here "Output" includes both output binding and pub, any suggestion for a better name?